filenames.gni

```diff
@@ -1,3 +1,5 @@
+import("//build/config/ui.gni")
```

Comment:
I'd prefer this file to be straight data and not contain any logic.

I see this has already been merged (sorry for not catching this earlier!), but would you be able to follow up and refactor this to expose the relevant data as e.g. `lib_sources_linux_x11` / `lib_sources_linux_ozone` and move the logic to `BUILD.gn`? This would follow the pattern already established in this file.

Reply:
Sure, sounds good. I'll follow up with a pull-request to refactor this like you're suggesting.

Reply:
Thanks!

Reply:
This should have been fixed in #26146. Let me know if you think further changes are necessary.